src/middlewared/middlewared/plugins/consulkv.py

```diff
@@ -34,6 +34,7 @@ class ConsulService(Service):
         """
         c = consul.aio.Consul()
         try:
+            logger.info('CONSUL ===> Add Key: {} Value: {}'.format(str(key), str(value)))
```

Comment:
Can we remove "CONSUL ===>" ?
Logger already has the name of the service stating it is the Consul service.